cheat-library/src/user/cheat/player/RapidFire.cpp

```diff
@@ -157,11 +157,14 @@ namespace cheat::feature
 		}
 		if (f_Randomize)
 		{
-			countOfAttacks = rand() % (f_maxMultiplier.value() - f_minMultiplier.value()) + f_minMultiplier.value();
-			return countOfAttacks;
+			if (f_minMultiplier.value() >= f_maxMultiplier.value()) 
+				countOfAttacks = f_minMultiplier.value();
+			else
+				countOfAttacks = rand() % (f_maxMultiplier.value() - f_minMultiplier.value() + 1) + f_minMultiplier.value();
+			return countOfAttacks - 1;
```

Comment:
```suggestion
			return countOfAttacks;
```
The formula already clamped value between min and max inclusive so no need to - 1

Reply:
As per my comment at second sentence https://github.com/Akebi-Group/Akebi-GC/pull/591#issue-1355221899 I'm proposing it to be change to user attack will execute exactly n times specified on the multiplier slider instead of user's first attack (1) + multiplier. I'm subtracting the first attack.

Reply:
It will behave like the video below with this PR

https://user-images.githubusercontent.com/78299538/187391606-6f817985-95d7-495c-8b5a-27c6684e7d07.mp4

Reply:
Ah okay, I see what you meant. I've always interpreted that as 'extra hits' since it is called 'multiplier' so was always counted the first hit together. You can dismiss mine suggestions or I'll remove mine later :upside_down_face: